marienfressinaud/FreshRSS
# Improvement: menu icons

Closes #3251
Address partially #3380

(this PR finishes  #3471, that I could not edit itself so I copied it and made the next steps )

Before:
![grafik](https://user-images.githubusercontent.com/1645099/143788013-a143224a-7a5d-4ead-b33b-6a96c8a24783.png)
Hover:
![grafik](https://user-images.githubusercontent.com/1645099/143788019-90b6dd16-ca07-4275-bf56-324147c8352b.png)

Theme Swage:
![grafik](https://user-images.githubusercontent.com/1645099/143788047-af365dce-e48e-4042-9064-7d175e883dd8.png)


After;
![grafik](https://user-images.githubusercontent.com/1645099/143787979-86bed543-3674-4d42-8791-e1c2aad3d50e.png)
Hover:
![grafik](https://user-images.githubusercontent.com/1645099/143787994-42c4a532-558b-4625-b275-31459f861089.png)

Thee Swage:
![grafik](https://user-images.githubusercontent.com/1645099/143788086-1983f69c-a122-415e-a656-dceb1ea02237.png)




Changes proposed in this pull request:

- Add title on user query menu item (For #3380)
- new icon for the sorting button
- new icon for the User Query button (used the icon from the User Query menu and deleted the plus symbol)
- some themes got an improvement while hovering over the "Bookmark current query"


How to test the feature manually:

- Hover the user query drop-down menu to see the tooltip



Pull request checklist:
